src/lib/sub-app/sub-app.factory.test.ts

```diff
@@ -20,6 +20,7 @@ describe('SubApp Factory', () => {
     const files: string[] = tree.files;
     expect(files.sort()).toEqual([
       '/nest-cli.json',
+      '/apps/jest-e2e.json',
```

Comment:
Can we move that file out of the "apps" directory (1 level up)?

Reply:
Resolved - Squashed change into https://github.com/nestjs/schematics/pull/1168/commits/6cac8f51326513183e05d5c0105d9e3f35ef233e.

Comparison of changes https://github.com/nestjs/schematics/compare/df3314d94882e26870b3a4689612df545ec9ff17..6cac8f51326513183e05d5c0105d9e3f35ef233e.